### PR TITLE
Small Tweaks for OSI Metric

### PR DESCRIPTION
**Pull request by Nebrethar**
A couple small syntax changes here as I work on the SPDX approved metric

Signed-off-by: Matt Snell <msnell@unomaha.edu>

### CODE REVIEW

File: metrics/OSI_Approved_Licenses.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ As from OSI: “Open source licenses are licenses that comply with the Open Sour
 
 ## Objectives
 
-Identify whether a project has licenses present which do not conform to the definition of open source. This transparency helps projects make conscious decisions about whether or not to include licenses that are not approved by OSI.
+Identify whether a project has licenses present which do not conform to the OSI definition of open source. This transparency helps projects make conscious decisions about whether or not to include licenses that are not approved by OSI.
```
Comment (GeorgLink) [inline metrics/OSI_Approved_Licenses.md:13]: okay

File: metrics/OSI_Approved_Licenses.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ Example: http://augur.osshealth.io/repo/Zephyr-RTOS/zephyr/risk
 
 Extract list of licenses from a code base, same as in License Coverage metric.
 Compare the list of licenses to [Licenses.json](https://raw.githubusercontent.com/spdx/license-list-data/master/json/licenses.json) and take note of how many licenses are approved by the OSI.
-Calculate the percentage of files on the OSI license list.
+Calculate the percentage of OSI approved files on the SPDX license list.
```
Comment (GeorgLink) [inline metrics/OSI_Approved_Licenses.md:33]: This doesn't make sense nor did the orginal sentence make sense.


Maybe this is better?
```suggestion
Calculate the percentage of files with an on the OSI approved license.
```

**Comment by germonprez** [comment]
@Nebrethar do you have a chance to address these changes?

**Comment by Nebrethar** [comment]
Moved to #116 with the reviews accepted

**Pull request closed without merging**